src/constants/FeatureFlags.js

```diff
@@ -0,0 +1,6 @@
+const FeatureFlags = {
+  REACT_DASHBOARD: 'react_dashboard',
+  REACT_CREATE: 'react_create',
+};
+
+export { FeatureFlags };
```

Comment:
Small nit pick: Do you think we'll use this constant somewhere without using the `useFeatureFlag` hook? If not I think we can put it together with the hook in the same file.

I think this `constants` folder was just added for constants that are used in multiple contexts, not related to one specific hook/component.

Reply:
Yep I think this can also live in useFeatureFlags

Reply:
Jep, good idea